### ISSUE TITLE
new (experimental) cve scan idea

### ISSUE THREAD

**Issue by cedwards** [HubbleStack/Nova#145]
I'd like to explore querying the vulners.com vulnerability API for our CVE scans. This means a step away from `openscap` and into a self-contained parser and comparison module. The benefits I see here (assuming vulners.com remains available) is that we can do real-time API queries (vs static hosted files), and it supports a wide range of Linux distributions.

What I imagine is a comparison of the `affectedPackage` list from the API return and the results of Salt's existing `pkg.list_pkgs`.

The `affectedPackage` includes (example):

```
"affectedPackage": [
{
"packageName": "icedtea-web",
"OS": "CentOS",
"operator": "lt",
"packageFilename": "icedtea-web-1.6.2-1.el6.x86_64.rpm",
"OSVersion": "6",
"packageVersion": "1.6.2-1.el6",
"arch": "x86_64"
},
```

I'd like to make use of the `operator` to define the comparison, and the `OSVersion` to check the correct version.

Beyond this basic comparison I'd like to report the `id` and the `title` along with the affected package(s).

**Comment by cedwards** [comment]
@thehoule - can you take a look?

**Comment by thehoule** [comment]
Yep!

**Comment by cedwards** [comment]
This was completed and will be merged today in #154 . Closing.
